### PR TITLE
Add Schema Header

### PR DESCRIPTION
**Pull request by Trenly**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/83817)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

**Comment by Trenly** [comment]
@denelon - Blocking-Issue #79677

**Comment by Trenly** [comment]
.

File: manifests/k/Kiwix/WikiMed/1.9.5.0/Kiwix.WikiMed.installer.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 # Created using wingetcreate
 
 PackageIdentifier: Kiwix.WikiMed
```
Comment (AmelBawa-msft) [inline manifests/k/Kiwix/WikiMed/1.9.5.0/Kiwix.WikiMed.installer.yaml:4]: InstallerUrl is not reachable, I think the manifest validation step is likely attempting to download the installer without success and reporting manifest-installer-validation-error label

**Comment by Trenly** [comment]
Bump

File: manifests/k/Kiwix/WikiMed/1.9.5.0/Kiwix.WikiMed.installer.yaml (hunk 1 above)
Comment (Trenly) [reply manifests/k/Kiwix/WikiMed/1.9.5.0/Kiwix.WikiMed.installer.yaml:4]: Will review

**Comment by stephengillie** [comment]
The Kiwix releases page only has v2.x.x packages. Since this version appears to no longer be hosted, should it be removed instead of having its header updated?

**Comment by Trenly** [comment]
@msftbot reset feedback

**Review by denelon — APPROVED** [review]

**Pull request merged**